Enabled user_allow_other

This allows Chromium OS to run FUSE-based file system daemons as a
non-root and non-chronos user, while allowing chronos to access the
mount file systems.

diff --git a/util/fuse.conf b/util/fuse.conf
index 3c9b6ac..c292f74 100644
--- a/util/fuse.conf
+++ b/util/fuse.conf
@@ -10 +10 @@
-#user_allow_other
+user_allow_other
